tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -37,14 +57,14 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
         case "Admin-0 capital":
         case "Admin-0 capital alt":
         case "Admin-0 region capital":
-          kind = "city";
+          kind = "locality";
```

Comment:
WARNING: Breaking v2 > v3 Protomaps schema change to converge on Tilezen kind values for `locality` (instead of `city`, which is still in `kind_detail`).